pkg/pgmodel/migrations/sql/1_base_schema.up.sql

```diff
@@ -307,38 +307,53 @@ CREATE TRIGGER make_metric_table_trigger
 
 -- Return a table name built from a full_name and a suffix.
 -- The full name is truncated so that the suffix could fit in full.
--- name size will always be exactly 63 chars.
+-- name size will always be exactly final_length chars.
 CREATE OR REPLACE FUNCTION SCHEMA_CATALOG.pg_name_with_suffix(
-        full_name text, suffix text)
+        full_name text, suffix text, final_length smallint)
```

Comment:
Not a huge fan of this being a variable. I think it might be better to do the macro like thing we're doing with schemas to replace that?